<ui version="4.0" >
 <comment>*********************************************************************
**
** Copyright (C) 1992-2006 Trolltech ASA. All rights reserved.
**
** This file is part of the Qt Assistant of the Qt Toolkit.
**
** This file may be used under the terms of the GNU General Public
** License version 2.0 as published by the Free Software Foundation
** and appearing in the file LICENSE.GPL included in the packaging of
** this file.  Please review the following information to ensure GNU
** General Public Licensing requirements will be met:
** http://www.trolltech.com/products/qt/opensource.html
**
** If you are unsure which license is appropriate for your use, please
** review the following information:
** http://www.trolltech.com/products/qt/licensing.html or contact the
** sales department at sales@trolltech.com.
**
** This file is provided AS IS with NO WARRANTY OF ANY KIND, INCLUDING THE
** WARRANTY OF DESIGN, MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE.
**
*********************************************************************</comment>
 <class>TabbedBrowser</class>
 <widget class="QWidget" name="TabbedBrowser" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>TabbedBrowser</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <widget class="QWidget" name="frontpage" >
      <attribute name="title" >
       <string>Untitled</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>8</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frameFind" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QToolButton" name="toolClose" >
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset resource="resources/images.qrc" >:/images/find_close.png</iconset>
        </property>
        <property name="autoRaise" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editFind" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>0</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolPrevious" >
        <property name="text" >
         <string>Previous</string>
        </property>
        <property name="icon" >
         <iconset resource="resources/images.qrc" >:/images/find_previous.png</iconset>
        </property>
        <property name="toolButtonStyle" >
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
        <property name="autoRaise" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolNext" >
        <property name="minimumSize" >
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text" >
         <string>Next</string>
        </property>
        <property name="icon" >
         <iconset resource="resources/images.qrc" >:/images/find_next.png</iconset>
        </property>
        <property name="toolButtonStyle" >
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
        <property name="autoRaise" >
         <bool>true</bool>
        </property>
        <property name="arrowType" >
         <enum>Qt::NoArrow</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkCase" >
        <property name="text" >
         <string>Case Sensitive</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkWholeWords" >
        <property name="text" >
         <string>Whole words</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelWrapped" >
        <property name="minimumSize" >
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>1050</width>
          <height>20</height>
         </size>
        </property>
        <property name="text" >
         <string>&lt;img src=":/images/find_wrap.png">&amp;nbsp;Search wrapped</string>
        </property>
        <property name="textFormat" >
         <enum>Qt::RichText</enum>
        </property>
        <property name="scaledContents" >
         <bool>true</bool>
        </property>
        <property name="alignment" >
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>81</width>
          <height>21</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources/images.qrc" />
 </resources>
 <connections/>
</ui>
